libfwupdplugin/fu-self-test.c

```diff
@@ -3800,6 +3744,7 @@ main(int argc, char **argv)
 	(void)g_setenv("FWUPD_LIBDIR_PKG", testdatadir, TRUE);
 	(void)g_setenv("FWUPD_SYSCONFDIR", testdatadir, TRUE);
 	(void)g_setenv("FWUPD_SYSFSFWATTRIBDIR", testdatadir, TRUE);
+	(void)g_setenv("FWUPD_SYSFSDMIDIR", testdatadir, TRUE);
```

Comment:
this new environment variable needs to be documented

Reply:
Fixed.